src/functions-and-arrays.js

```diff
@@ -1,73 +1,488 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else if (num1 < num2) {
+    return num2;
+  } else {
+    return num1, num2;
+  }
+}
+
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = [
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
+];
+
+function findLongestWord(array) {
+  if (array.length === 0) {
+    return null;
+  } else if (array.length == 1) {
```

Comment:
this check for " array.length == 1, return the single word " is actually redundant. You can see that if you remove it everything checks ok and this is because your for loop will also check if the array only has a single word without any issue. Also, remember to use "===" instead of "==" for comparisons unless you actually want to convert data types for some reason ( which usually you won't even do with "==" )